### PR TITLE
ADCM-1554 add allure attach steps for configs and hostcomponents

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: adcm_client/objects.py [hunk 1]
```diff
@@ -279,6 +279,7 @@ def config(self, full=False):
     def config_set(self, data):
         # this check is incomplete, cases of presence of keys "config" and "attr" in config
         # are not considered
+        allure_attach_json(data, name="Completed config")
```
Comment (dgusakov) [inline adcm_client/objects.py:282]:
```suggestion
        allure_attach_json(data, name="Complete config")
```

**Review by dgusakov — APPROVED** [review]

File: adcm_client/objects.py [hunk 2]
```diff
@@ -301,6 +302,7 @@ def update(d, u):
             return d
         # this check is incomplete, cases of presence of keys "config" and "attr" in config
         # are not considered
+        allure_attach_json(data, name="Changed fields")
         is_full = "config" in data and "attr" in data
         config = self.config(full=is_full)
```
Comment (dgusakov) [inline adcm_client/objects.py:307]:
```suggestion
        config = self.config(full=is_full)
        allure_attach_json(config, name="Original config")
```

Comment (skhomuti) [reply adcm_client/objects.py:307]: original config will be attached in `config_set` function

Comment (skhomuti) [reply adcm_client/objects.py:307]: fix

File: adcm_client/objects.py (hunk 1 above)
Comment (skhomuti) [reply adcm_client/objects.py]: fix

**Review by dgusakov — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Pull request merged**